app/models/collection.rb

```diff
@@ -1,5 +1,6 @@
 class Collection < ApplicationRecord
   include ActiveModel::ForbiddenAttributesProtection
+  include Filterable
```

Comment:
Can you add a test for Collection being filterable?

https://github.com/otwcode/otwarchive/blob/bedbbef8c43585d97b5a56062355921e4c8f5fa7/spec/models/concerns/filterable_spec.rb#L197-L205